### ISSUE TITLE
Question on the Commits and Releases discrepancy

### ISSUE THREAD

**Issue by DevTheFool** [proxidize/proxidize-android#92]
For every release, there is a bug fix, but if you look at what changed (the commits), only the README.md file is changing every time.
Why is that? Is the code in the GitHub the same as in the .apk?

Example: 
[release](https://github.com/proxidize/proxidize-android/releases/tag/v2.1.6), [commit](https://github.com/proxidize/proxidize-android/compare/v2.1.6...main)
[release](https://github.com/proxidize/proxidize-android/releases/tag/v2.1.5), [commit](https://github.com/proxidize/proxidize-android/compare/v2.1.5...main)
...

**Comment by AbedElez** [comment]
Hello,

The latest code is in the dev branch. We haven't merged as we stopped production a while back. We plan to continue the project soon under a new name.

**Comment by DevTheFool** [comment]
@AbedElezz Hi, thanks for the response and happy Christmas.
I am not sure if I understand what you mean. This [dev](https://github.com/proxidize/proxidize-android/tree/dev) branch with only 2 commits 10 months ago? But then where are the code changes for the releases that are newer, like 6 months ago? The tags in the releases point to a commit in the main branch with only a README.md change. Where can I find the actual code that was used to make the APKs in the latest releases? This, plus the result on Virus Total, is really suspicious.
![image](https://github.com/proxidize/proxidize-android/assets/110111354/002ff983-8b17-4357-a86a-6b0a7e9d22ca)
I really like this application and I hope it's legit, but I can't install it until I know for sure it's not a virus. Can you please help me understand why the releases are not showing what was changed and where to find the actual code up to date code used in the latest [release](https://github.com/proxidize/proxidize-android/releases/tag/v2.1.6)?

**Comment by AbedElez** [comment]
This project is a very low priority for us at the moment. Hence, why we haven't uploaded the code base as it takes a lot of time to detangle from internal codebase systems before we can make it public.

This will change early next year as we plan a revamp of the project.

In regards to the VirusTotal, please revert to this: https://github.com/proxidize/proxidize-android/issues/7
